Sources/Routing/Views/RoutingView.swift

```diff
@@ -21,7 +21,7 @@ public struct RoutingView<RootView: View, Routes: ViewDisplayable>: View {
                     $0.viewToDisplay
                         .environmentObject(router)
                 }
-                .sheet(item: $router.sheet) {
+                .sheet(item: $router.sheet, onDismiss: router.onDismiss) {
                     $0.viewToDisplay
```

Comment:
We should also add the `onDismiss: router.onDismiss` to the fullScreenCover view modifier